tests/Unit/Instrumentation/AwsSdk/AwsSdkInstrumentationTest.php

```diff
@@ -37,6 +55,55 @@ public function testInstrumentationInit()
         );
     }
 
+    public function testSetXrayPropagator()
+    {
+        $this->assertInstanceOf(
+            Propagator::class,
+            new Propagator()
+        );
+    }
+
+    public function testGetXrayPropagator()
+    {
+        $propagator = new Propagator();
+        $this->awsSdkInstrumentation->setPropagator($propagator);
+
+        $this->assertSame(
+            $this->awsSdkInstrumentation->getPropagator(),
+            $propagator
+        );
+    }
+
+    public function testSetTracerProvider()
+    {
+        $this->awsSdkInstrumentation->setTracerProvider($this->tracerProvider);
+
+        $this->assertInstanceOf(
```

Comment:
useless assertion. you should be checking the provider you get back from instrumentation